src/main/fc/settings.c

```diff
@@ -633,6 +633,9 @@ const clivalue_t valueTable[] = {
     { "osd_cap_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 20000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, cap_alarm) },
     { "osd_alt_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 10000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, alt_alarm) },
 
+    { "osd_ah_max_pit",             VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 900 }, PG_OSD_CONFIG, offsetof(osdConfig_t, ahiMaxPitch) },
+    { "osd_ah_max_rol",             VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 900 }, PG_OSD_CONFIG, offsetof(osdConfig_t, ahiMaxRoll) },
```

Comment:
Sorry for the late comment, but we don't need 0.1 degrees of precision for these. So they could be stored in a `uint8_t` and have range 0 to 90. They will then need to be multiplied by 10 at point of usage.